cmd/girafSort/girafSort_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/giraf"
+	"testing"
+)
+
+func TestGirafSort(t *testing.T) {
+	graphFile := "../../sort/testdata/mini.gg"
+	readsFile := "../../sort/testdata/miniReads.giraf"
+	outFile := "miniReads_sorted_test.giraf"
+	topoOrder := girafSort(readsFile, graphFile, 100, outFile)
+	actual := giraf.Read(outFile)
+	relabelIdBySortOrder(topoOrder, actual)
+
+	var lastId uint32
+	for i := range actual {
+		if i == 0 {
+			lastId = actual[i].Path.Nodes[0]
+		}
+		if actual[i].Path.Nodes[0] < lastId {
```

Comment:
Should "<" be "<=" since ids should be unique?

Reply:
duplicate Ids are allowed. This is sorting reads aligned to a graph, but not the graph itself. Mulitple reads can align to the same node.